assemblies/monitoring-data-drift.adoc

```diff
@@ -13,7 +13,10 @@ As a data scientist, you might want to monitor your deployed models for data dri
 
 You can use data drift monitoring metrics from TrustyAI in {productname-long} to provide a quantitative measure of the alignment between the training data and the inference data. 
 
-For information about the specific data drift metrics, see link:https://opendatahub.io/docs/monitoring-data-science-models/#supported-drift-metrics_drift-monitoring[Supported drift metrics].
+ifdef::upstream[]
```

Comment:
- You don't need the ifdef::upstream[] since this assembly is only used upstream.
- Use the attribute in link:{odhdocshome}/monitoring-data-science-models/#supported-drift-metrics_drift-monitoring[Supported drift metrics].